overlord/devicestate/devicestatetest/state.go

```diff
@@ -32,3 +34,19 @@ func Device(st *state.State) (*auth.DeviceState, error) {
 func SetDevice(st *state.State, device *auth.DeviceState) error {
 	return internal.SetDevice(st, device)
 }
+
+// MarkInitialized flags the state as seeded and the device registered to avoid
+// running the seeding code etc in tests, it also tries to sets the model to
+// generic-classic.
+// If the initial model is imporant this cannot be used.
+func MarkInitialized(st *state.State) {
```

Comment:
AIUI this is used for tests only? If so I wonder if we should give it a stronger test-only name. Something like `MockMarkInitialized`, `MarkInitializedForTests` or similar?

Reply:
it's inside devicestatetest, this is a test helper only package to start with

Reply:
Uh, silly me, ignore me then